airflow/providers/openai/hooks/openai.py

```diff
@@ -77,6 +89,165 @@ def get_conn(self) -> OpenAI:
             **openai_client_kwargs,
         )
 
+    def create_chat_completion(
+        self,
+        messages: list[
+            ChatCompletionSystemMessageParam
+            | ChatCompletionUserMessageParam
+            | ChatCompletionAssistantMessageParam
+            | ChatCompletionToolMessageParam
+            | ChatCompletionFunctionMessageParam
+        ],
+        model: str = "gpt-3.5-turbo",
+        **kwargs: Any,
+    ) -> ChatCompletionMessage:
+        """
+        Create a model response for the given chat conversation and returns the message.
+
+        :param messages: A list of messages comprising the conversation so far
+        :param model: ID of the model to use
+        """
+        response = self.conn.chat.completions.create(model=model, messages=messages, **kwargs)
+        return response.choices[0].message
```

Comment:
Will the response always contain a choices list with one value?

Reply:
It's a good question.  I've not seen it and this is based upon OpenAI's official [docs](https://platform.openai.com/docs/api-reference/chat).

Reply:
```

Reply:
I could set `n=1` as a default argument which would return `response.choices[0].message` of type `Message` but if this was overwritten it could return `response.choices` which would be `list[Message]` but I'm not sure if that would be a good practice or not.

Reply:
@Lee-W Do you have any thoughts on this?

Reply:
I feel like we probably should return `list[Message]`. Will that cause any issue if we do so?

Reply:
I suppose not.  My initial thought was that I just wanted to return the message object but, if there can be more than one, then returning the list and then handling this outside probably seems like the correct thing to do.